pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -454,15 +450,15 @@ func waitLogs(ctx context.Context, id string, w *fsnotify.Watcher, runtimeServic
 		case e := <-w.Events:
 			switch e.Op {
 			case fsnotify.Write:
-				return true, false, nil
-			case fsnotify.Create:
 				fallthrough
 			case fsnotify.Rename:
-				fallthrough
+				return true, false, nil
 			case fsnotify.Remove:
 				fallthrough
 			case fsnotify.Chmod:
-				return true, true, nil
+				fallthrough
+			case fsnotify.Create:
+				return true, filepath.Base(e.Name) == logName, nil
```

Comment:
Please tell me about the concept of the return value for each notify.
I thought that the return value should be `return true, true, nil` only when the fsnotify.Create of the current log file (filepath) is notified.
Why do fsnotify.Remove and fsnotify.Chmod also need to `return true,true,nil`?

Reply:
You are right. There is no necessity to return true when  fsnotify.Remove and fsnotify.Chmod .

Reply:
You are right. There is no necessity to return true when fsnotify.Remove and fsnotify.Chmod.